### PR TITLE
Docs Meshsync CRD

### LINKED ISSUE #9294: [Docs] Meshery CRD Reference: MeshSyncs
### Current State

The Meshery CRDs are not included in Meshery Docs.

### Desired State
Create a "CRDs Reference" page in the "Reference" section of Meshery Docs. Include the following information:

# CRD Name
_description of the CRD's purpose_

## YAML Synopsis

```
apiVersion: core.meshery.io....
kind: MeshSync
metadata:
  name: 
spec:
.
.
.
```

## Fields

- **apiVersion** – API version being used. Must be v1alpha3 or later for this syntax.
- **kind** – Resource type. Must be set to MeshSync
- **metadata** -
- **name** – 
- **spec** -
  -

- **property** – description
- **property** – description
- **property** – description
- **property** – description

## Usage

_sample use scenario_

## Example Use

_explanaation of example_

```yaml
apiVersion: core.meshery.io....
kind: MeshSync
metadata:
  name: 
spec:
.
.
.

```

_explanaation of example_

## See Also
_link to other Meshery CRD_

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by KiptoonKipkurui**
**Notes for Reviewers**

This PR fixes #9294

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 
Documentation for Meshsync CRD

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 83eaff8264c77b73e29809e8b761732bb97200a4 at: https://65646c63e167cf28c853068f--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit b1cc901d66710b479661aa7e0c130e5a56fb4f96 at: https://656494b4d6402946c681ab60--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 644086886b8f9c669c64eae462ce21b21ec4f937 at: https://65649632fdb1c345ec781b30--meshery-docs-preview.netlify.app

**Review by leecalcote — COMMENTED** [review]
@KiptoonKipkurui I wonder if we might be able to get a bit more around the ConfigMap...

File: docs/pages/reference/meshync-crd-reference.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+title: MeshSync CRD Reference
+abstract: "Meshync CRD Documentation and Reference"
+permalink: reference/meshync-crd
+redirect_from: reference/
+type: Reference
+language: en
+---
+
+# MeshSync CRD
+The MeshSync CRD is used as a configuration tool the the Meshsync service. The CRD is used to control the replica count for Meshsync instances, broker configuration and the resources that Meshsync watches and listens to in a cluster.
```
Comment (MUzairS15) [inline docs/pages/reference/meshync-crd-reference.md:12]:
```suggestion
The MeshSync CRD is used as a configuration tool for the MeshSync service. The CRD is used to control the replica count for MeshSync instances, broker configuration and the resources that MeshSync watches and listens to in a cluster. 
```

Comment (MUzairS15) [reply docs/pages/reference/meshync-crd-reference.md:12]: MeshSync service? I think this will create confusion. Can you describe more explicitly.

Comment (leecalcote) [reply docs/pages/reference/meshync-crd-reference.md:12]: Instead, just say “MeshSync”, yeah?  Also, a link to the existing Alesha sync page can help explain MeshSync’s purpose and overall behavior.

**Comment by l5io** [comment]
🚀 Preview for commit 205712b4d8e96e50c5616cc29678f3795ce3fa17 at: https://6566494622498310f463e713--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit fea923a37c48d910d09a47a0d243a747422f443f at: https://65677188e0005a4130168de5--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit f61116d85d7a448d622d1359e380cc8c0ef357a1 at: https://6567732ae50eab423f9a016a--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit a70c57ea04e595fc5f9ea8d3e45b1ce238b796ff at: https://6567880a3970d150aa42ef1d--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 78cdf598f690b16255ade7045b70d51f2b7c1d5d at: https://65678a9dfe8d9500ac5ee506--meshery-docs-preview.netlify.app

**Comment by KiptoonKipkurui** [comment]
@leecalcote @MUzairS15 I have made the relevant changes, more review would be appreciated

**Review by leecalcote — APPROVED** [review]
👨‍🎤 🎸 @KiptoonKipkurui 🥇

**Pull request merged**